include/config/config_game.h

```diff
@@ -1,4 +1,19 @@
 #ifndef CONFIG_GAME_H
 #define CONFIG_GAME_H
 
+// Configure text speed (1 is vanilla)
+#define TEXT_SPEED 2
```

Comment:
I'd prefer we call this something like `DIALOGUE_PRINT_SPEED` instead since "text" could be potentially too generic.